.gitignore

```diff
@@ -6,3 +6,4 @@ vendor/
 __pycache__/
 /duffle
 .vscode/settings.json
+.idea
```

Comment:
There is a school of thought that developers should be responsible for ignoring their own IDE's settings in a global gitignore file so as not to pollute the gitignore files of multiple projects. However, this change seems worthwhile because goland (and vscode) are so prevalent.

Reply:
I agree with this school of thought 👍